app/jobs/content_restored_version_event_job.rb

```diff
@@ -1,26 +1,13 @@
-class ContentRestoredVersionEventJob < EventJob
+# A specific job to log a file restored version to a user's activity stream
+class ContentRestoredVersionEventJob < ContentEventJob
   attr_accessor :revision_id
 
   def initialize(generic_file_id, depositor_id, revision_id)
-    self.generic_file_id = generic_file_id
-    self.depositor_id = depositor_id
+    super(generic_file_id, depositor_id)
     self.revision_id = revision_id
```

Comment:
This isn't a change you've made, but I wonder if this should be `@revision_id` instead of `self.revision_id` to be more idiomatic.